src/applications/edu-benefits/10203/content/directDeposit.jsx

```diff
@@ -11,7 +11,7 @@ export const directDepositDescription = () => {
         Scholarship, we’ll need to know where to deposit the funds.
       </p>
       <img
-        src="/img/direct-deposit-check-guide.png"
+        src="/img/direct-deposit-check-guide.svg"
```

Comment:
This relative path references the image in the content-build repo bucket when deployed to dev/staging/prod. Instead of a relative path, please use an absolute path. You can import the bucket URL from here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/constants/buckets.js#L7. That will reference the image you're adding this PR.

Reply:
Or, what Tim said already, haha.